### DISCUSSION TITLE
Shared List with wife / partner ...

### DISCUSSION THREAD

**Discussion by Markus-** [Kovah/LinkAce#385] [General]
Hi,

I just stumbled over this awesome looking tool while searching for a bookmark manager.
So I have the following use case:

My wife and I have private link collections (work, hobby, personal stuff) and links I want to manage but my wife needs to access regulary (bank accounts, internal wiki, manual links for stuff). Is there a way to share a link with another user but don't make it public? 

Or is there only the distinction between public and private for lists / links?

Regards
markus

**Comment by Kovah** [comment]
Hi!
Currently, lists and so on can only be made public or private. But there already is a feature request for sharing lists with a private link. That would be a possible solution for your use case. What do you think?
- #146

**Comment by Markus-** [comment]
Probably an owner / maintainer permission for a list is a more appropiate way of implementing this. With two users this can work, but I am thinking about managing a list of internal applications for my team so a deputy could be useful. For the beginning your Feature request would work, but I like the idea of be able to manage a list with several users.

**Comment by Kovah** [comment]
So, instead of having a secret link you can share with someone, you would like to explicitly grant permissions on a user-level?

**Reply by Markus-** [reply]
Kind of. So probably like a Github-Project where you have an owner, a maintainer and a viewer?

**Reply by Kovah** [reply]
Alright, got it. This will, however, only be possible when official multi-user support has landed in LinkAce. It is currently planned for version 2, which will likely not happen this year anymore.
But I guess the secret link could be added earlier as a workaround.

**Reply by Markus-** [reply]
OK. Thanks.
